packages/icons/src/PentahoIconWrapper.tsx

```diff
@@ -0,0 +1,90 @@
+import styled from "@emotion/styled";
+import { SizeProp } from "@fortawesome/fontawesome-svg-core";
+import {
+  FontAwesomeIcon,
+  FontAwesomeIconProps,
+} from "@fortawesome/react-fontawesome";
+import { getColor, HvColorAny, HvSize } from "@hitachivantara/uikit-styles";
+
+import { IconSize, svgSizeMap } from "./IconBase";
+
+export interface PentahoIconWrapperProps
+  extends Omit<FontAwesomeIconProps, "color" | "size"> {
+  size?: HvSize;
+  iconSize?: IconSize;
```

Comment:
If we're moving to the `size` prop, there's shouldn't be an `iconSize`

We could do `HvSize | IconSize` for increased compatibility though. Also, allowing `number` would be nice

```suggestion
  size?: HvSize | IconSize | number;
```

Reply:
I agree, I will add number when you pr with icon base sizes enters so I can use the same logic.